Description: Support Linux 2.6.36 and older
 Linux 2.6.37 bumps EV_VERSION to 0x010001. To continue support for
 Linux 2.6.36 and older we explicitely allow EV_VERSION 0x010000
 here. This should not be a problem since tslib does not currently use
 any of the API that was changed in this bump (upstream commit
 8613e4c2872a87cc309a42de2c7091744dc54d0e).
Author: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Last-Update: 2011-06-07
Forwarded: no
Bug-Debian: http://bugs.debian.org/623095

--- a/plugins/input-raw.c
+++ b/plugins/input-raw.c
@@ -60,7 +60,7 @@
 	unsigned long absbit[ABS_MAX / BITS_PER_LONG + 1];
 
 	if (! ((ioctl(ts->fd, EVIOCGVERSION, &version) >= 0) &&
-		(version == EV_VERSION) &&
+		(version == EV_VERSION || version == 0x010000) &&
 		(ioctl(ts->fd, EVIOCGBIT(0, sizeof(bit)), bit) >= 0) &&
 		(bit[0] & (1 << EV_ABS)) &&
 		(ioctl(ts->fd, EVIOCGBIT(EV_ABS, sizeof(absbit)), absbit) >= 0) &&
